src/css/main.css

```diff
@@ -71,6 +71,60 @@ button:hover{
     border:1px solid #f00;
 }
 
+.form-row {
+    display: flex;
+    width: "50%";
+}
+
+.form-row-extremes {
+    display: inline-block; 
+    width: 100%;
+}
+
+.form-group { 
+    flex:1;
+    padding-left:5px;
+    padding-right:5px;
+    padding-bottom: 0;
+}
+.form-group-left {
+    float: left;
+    padding-left:5px;
+    padding-right:5px;
+    padding-bottom: 0; 
+}
+.form-group-right {
+    float: right;
+    padding-left:5px;
+    padding-right:5px;
+    padding-bottom: 0; 
+}
+.form-group-center {
+    flex:1;
+    padding-left:5px;
+    padding-right:5px;
+    padding-bottom: 0;
+}

```

Comment:
I don't think this is being used right now. If you have a plan for how this will be used later feel free to keep it around, but  if it ends up not being used we should remove.